src/components/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoListItem from './TodoListItem';
+
+const TodoList = ({
+  todoList, onRemoveTodo, isChecked, handleChange,
+}) => {
+  
+  return (
+    <ul>
+      {todoList.map((todo) => (
+        <TodoListItem
+          key={todo.id}
+          todo={todo}
+          onRemoveTodo={onRemoveTodo}
+          isChecked={isChecked[todo.id]}
+          handleChange={() => handleChange(todo.id)}
+        />
+      ))}
```

Comment:
sometimes in an example like this, you might want to both have access to the entire `todo` object, but also want to destructure its `id` so you can use it more easily. In that case you can leave the object named in the function argument (or prop) and then destructure inside the body of the function. You'd have to switch the arrow function to have an explicit `{}` code block though: 

```suggestion
      {todoList.map((todo) => {
        const { id } = todo;  
      
        return <TodoListItem
          key={id}
          todo={todo}
          onRemoveTodo={onRemoveTodo}
          isChecked={isChecked[id]}
          handleChange={() => handleChange(id)}
        />
      })}
```

Reply:
Updated!